sdk/iot/hub/samples/src/generate_validation.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+echo "Generating Validation Cert"
```

Comment:
This should be removed - it's too complicated and not the recommended way.
For Intermediate CAs we can mention that KeyValut can support generating and manage the PKI.

Reply:
using self-signed now